### PR TITLE
ActivityFinder: Add search index config for SOLR backend.

### PR DESCRIPTION
**Pull request by afi13**
Adds the ability to select search_api index for Activity Finder SOLR backend in the AF settings form.

## Steps for review

- [ ] Go to `/admin/openy/settings/activity-finder`
- [ ] Verify the admin is able to select the search api index for Activity Finder SOLR backend.
- [ ] Create a landing with Activity Finder paragraph and make sure everything works fine.

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer CI. <br/>CI code sniffer errors
- [ ] All tests are running and there are no failed tests reported by CI. <br/>Behat test results
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build1865/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1865/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build1865/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build1865/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build1865/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build1865/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build1865/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build1865/

**Review by hamrant — DISMISSED** [review]

File: modules/custom/openy_activity_finder/config/install/openy_activity_finder.settings.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 backend: openy_activity_finder.solr_backend
+index: default
```
Comment (podarok) [inline modules/custom/openy_activity_finder/config/install/openy_activity_finder.settings.yml:2]: Should we implement hook_update_N for those who already uses previous revision of this config?
I guess - yes. Could you make it, please?

**Review by podarok — CHANGES_REQUESTED** [review]
Let's maintain upgrade path

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build1867/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1867/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build1867/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build1867/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build1867/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build1867/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build1867/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build1867/

File: modules/custom/openy_activity_finder/config/install/openy_activity_finder.settings.yml (hunk 1 above)
Comment (afi13) [reply modules/custom/openy_activity_finder/config/install/openy_activity_finder.settings.yml:2]: @podarok Fixed, tnx.

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build1867/

**Review by podarok — DISMISSED** [review]

**Comment by podarok** [comment]
There are conflicts, could you fix @afi13 ?

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build1879/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1879/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build1879/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build1879/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build1879/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build1879/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build1879/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build1880/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1880/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build1880/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build1880/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build1880/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build1880/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build1880/blogWCAG2AAhtmlcs.txt

**Comment by afi13** [comment]
@podarok conflicts have been solved.

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build1879/

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build1880/

**Review by hamrant — APPROVED** [review]

**Pull request merged**